var/spack/repos/builtin/packages/alquimia/package.py

```diff
@@ -0,0 +1,77 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Alquimia(CMakePackage):
+    """Alquimia is an interface that exposes the capabilities
+    of mature geochemistry codes such as CrunchFlow and PFLOTRAN"""
+
+    homepage = "https://github.com/LBL-EESA/alquimia-dev"
+
+    version('xsdk-0.2.0', git='https://github.com/LBL-EESA/alquimia-dev.git', tag='xsdk-0.2.0')
+    version('develop', git='https://github.com/LBL-EESA/alquimia-dev.git')    
+
+    variant('shared', default=True,
+            description='Enables the build of shared libraries')
+    variant('debug',  default=False,
+            description='Builds a debug version of the libraries')
+
+    depends_on('mpi')
+    depends_on('hdf5')
+    depends_on('pflotran@xsdk-0.2.0', when='@xsdk-0.2.0')
+    depends_on('pflotran@develop', when='@develop')    
+    depends_on('petsc@xsdk-0.2.0', when='@xsdk-0.2.0')
+    depends_on('petsc@develop', when='@develop')    
+
+    def cmake_args(self):
+        spec = self.spec
+
+        options = ['-DCMAKE_C_COMPILER=%s' % self.spec['mpi'].mpicc,
```

Comment:
you may as well join this with the next lines as
```
options = [
   '-DCMAKE_....',
   ...
]
```
also you set `spec= self.spec` above, but don't use it right here.